ibc-clients/ics07-tendermint/src/consensus_state.rs

```diff
@@ -43,6 +43,15 @@ impl ConsensusState {
     }
 }
 
+/// Can't use [`Infallible`](core::convert::Infallible) because TryFrom<Error = ClientError> is required by
+/// [`ConsensusStateDecoder`](ibc_core_client::context::consensus_state::ConsensusStateDecoder).
+impl TryFrom<ConsensusState> for ConsensusStateType {
+    type Error = ClientError;
+    fn try_from(value: ConsensusState) -> Result<Self, Self::Error> {
+        Ok(value.0)
+    }
+}
```

Comment:
This can be `impl From<ConsensusState>` after #1203 is merged.

Reply:
[082dc18](https://github.com/cosmos/ibc-rs/pull/1202/commits/082dc181b343335bc474300a2cfad817c0587c81)